### ISSUE TITLE
Large loss with pretrained_model trained on CASIA-WebFace_align

### ISSUE THREAD

**Issue by Victoria2333** [davidsandberg/facenet#829]
I run:
python src/train_softmax.py --data_dir /home/han/facenet/CASIA-WebFace_align --image_size 160 __embedding_size 512 /home/han/facenet/src/pretrained_models/20180408-102900/model-2018408-102900.ckpt-90
Theoretically, the accuracy of pretrained model trained 90 epoch should be nearly 0, but when I trained on the pretrained model on david's dataset-CASIA-WebFace, the loss is 1.198,
and when i add all parameters except validation parameters followed https://github.com/davidsandberg/facenet/wiki/Classifier-training-of-inception-resnet-v1:
python src/train_softmax.py --logs_base_dir /home/han2/logs/facenet/ --models_base_dir /home/han2/models/facenet/ --data_dir /home/han/facenet/CASIA-WebFace_align --image_size 160 --model_def models.inception_resnet_v1 --optimizer ADAM --learning_rate -1 --max_nrof_epochs 150 --keep_probability 0.8 --random_crop --random_flip --use_fixed_image_standardization --learning_rate_schedule_file data/learning_rate_schedule_classifier_casia.txt --weight_decay 5e-4 --embedding_size 512 --pretrained_model /home/han2/facenet/src/pretrained_models/20180408-102900/model-20180408-102900.ckpt-90
the loss is 2.775 which is very large...
Has anybody else tried this??? I couldn't figure out the reason, is the code wrong???
